app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/MediaAdapter.kt

```diff
@@ -462,15 +463,24 @@ class MediaAdapter(
     private fun askConfirmDelete() {
         val itemsCnt = selectedKeys.size
         val firstPath = getSelectedPaths().first()
-        val items = if (itemsCnt == 1) {
-            "\"${firstPath.getFilenameFromPath()}\""
+        val fileDirItem = File(firstPath).toFileDirItem(activity)
+        val size = fileDirItem.getProperSize(activity, countHidden = true).formatSize()
+        val itemsAndSize = if (itemsCnt == 1) {
+            "\"${firstPath.getFilenameFromPath()}\" ($size)"
         } else {
-            resources.getQuantityString(R.plurals.delete_items, itemsCnt, itemsCnt)
+            val paths = getSelectedPaths()
+            val fileDirItems = ArrayList<FileDirItem>(paths.size)
+            paths.forEach {
+                val fileDirItem = File(it).toFileDirItem(activity)
+                fileDirItems.add(fileDirItem)
+            }
+            val size = fileDirItems.sumByLong { it.getProperSize(activity, countHidden = true) }.formatSize()
+            "${resources.getQuantityString(R.plurals.delete_items, itemsCnt, itemsCnt)} ($size)"
```

Comment:
move this resource... string out to a local variable above please, this is way too much stuff within brackets

Reply:
Created a variable for it ("deleteItemsString")